exercises/supervised-classification/ex_rnw/sol_hyperplanes.Rnw

```diff
@@ -0,0 +1,41 @@
+A hyperplane in 2D is just a line. We know that two points are sufficient to describe a line, 
+so all we need to do is pick two points fulfilling the hyperplane equation.
+
+\begin{itemize}
+\item $\theta_0 = 0, \theta_1 = \theta_2 = 1$ $\rightsquigarrow$ e.g., (0, 0) and (1, -1).
```

Comment:
- du meinst hier, dass (x1, x2) = (0,0) bzw. (x1,x2) = (1,-1) oder? Dann schreib das besser noch dazu
- im Plot steht dann x und y, das kommt aber in der Angabe gar nicht vor => besser angleichen?